packages/acs-ui-common/src/removeImageTags.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/**
+ * @internal
+ */
+export const removeImageTags = (event: { content: DocumentFragment }): void => {
```

Comment:
As this function is moved to common packlet, we can reuse it in some places in Storybook too. 

The only thing that it shouldn't be updated for Docs tab for RTE send box and Message thread like the one on this page (as Contoso won't see an example of how to implement this`removeImageTags` function)
![image](https://github.com/user-attachments/assets/ca3aa08e-b4a1-48d2-be2e-3c54ed9aabf1)